MaxMind.MinFraud/Request/TransactionReport.cs

```diff
@@ -83,6 +90,15 @@ public sealed class TransactionReport
             string? transactionId = null
         )
         {
+            if (ipAddress == null && minfraudId == null && string.IsNullOrEmpty(maxmindId)
```

Comment:
It might be worth checking that `minfraudId` is not the 0 UUID too. I think this can be done with `minfraudId.Empty`. Maybe we should only set it if it is non-empty too.